src/ovs/aca_ovs_l3_programmer.cpp

```diff
@@ -831,7 +851,10 @@ int ACA_OVS_L3_Programmer::create_or_update_l3_neighbor(
     ACA_LOG_DEBUG("router ID:%s\n ", router_it->first.c_str());
     // try to see if the destination subnet GW is connected to the current router
     auto found_subnet = router_it->second.find(subnet_id);
-
+    for (auto kv : router_it->second) {
+      ACA_LOG_INFO("[create_or_update_l3_neighbor] router ID: [%s], subnet routering table's subnet ID: [%s], subnet_id we're looking for: [%s]\n",
```

Comment:
Please change this log to `ACA_LOG_DEBUG`

Reply:
Fixed